simple_ldap_user/simple_ldap_user.module

```diff
@@ -480,8 +480,17 @@ function simple_ldap_user_sync_user_to_drupal($drupal_user) {
   // Load the LDAP user, force a cache reset.
   $ldap_user = SimpleLdapUser::singleton($drupal_user->name, TRUE);
 
-  // Nothing to sync.
-  if (!$ldap_user->exists || (isset($drupal_user->uid) && $drupal_user->uid == 1)) {
+  // Nothing to sync. Check if user is an admin
```

Comment:
The comment 'check if user is an admin' really applies only to the interior of this first block, not to the whole conditional block.  It should move inside the first block.